src/ChipArrayField/ChipArrayField.jsx

```diff
@@ -0,0 +1,209 @@
+    label={value.value}
+  />
+)
+
+export const StyledChipInput = withStyles(styles)(ChipInput)
+
+/**
+ * The `<ChipArrayField>` component allows the editing and validation of a
+ * collection of items.
+ *
+ * It is designed to be a better experience when things like email addresses by
+ * preventing the need to comma separate the addresses manually.
+ *
+ * It is under the hood of EmailArrayField, but without some email specific features
+ *
+ * ~~~js
+ * import { ChipArrayField } from '@theconversation/ui'
+ *
+ * <ChipArrayField entries={["First"]} />
+ * ~~~
+ */
+class ChipArrayField extends React.Component {
+  constructor () {
+    super()
+    this.state = {
+      entries: []
+    }
+    this.handleAddEntry = this.handleAddEntry.bind(this)
+    this.handleDeleteEntry = this.handleDeleteEntry.bind(this)
+    this.sortEntries = this.sortEntries.bind(this)
+    this.validateEntries = this.validateEntries.bind(this)
+  }
+
+  componentDidMount () {
+    const { entries } = this.props
+    this.setState({
+      entries
+    })
+  }
+
+  handleAddEntry (value) {
+    let { entries } = this.state
+    entries.push(value)
+    this.setState({ entries })
+  }
+
+  handleDeleteEntry (entry, index) {
+    let { entries } = this.state
+    entries.splice(index, 1)
+    this.setState({ entries: entries })
+  }
+
+  sortEntries (entries) {
+    if (this.props.onSortEntry) {
+      return entries.concat().sort(this.props.onSortEntry)
+    }
+    return entries.concat().sort((a, b) => {
+      const valA = a.value.toLowerCase()
+      const valB = b.value.toLowerCase()
+      return (valA < valB) ? -1 : (valA > valB) ? 1 : 0
+    })
+  }
+
+  validateEntries (entries) {
+    return entries.map((e) => this.validateEntry(e))
+  }
+
+  validateEntry (e) {
+    let isValid = true
+    if (this.props.onValidateEntry) {
+      isValid = this.props.onValidateEntry(e)
+    }
+    return { value: e, valid: isValid }
+  }
+
+  render () {
+    let entries = this.validateEntries(this.state.entries)
+    if (this.props.sorted) {
+      entries = this.sortEntries(entries)
+    }
+    return (
+      <StyledChipInput
+        id={this.props.id}
+        variant={'outlined'}
+        helperText={this.props.helperText}
+        label={this.props.label}
+        disabled={this.props.disabled}
+        value={entries} // for now
+        onAdd={(entry) => this.handleAddEntry(entry)}
+        onDelete={(entry, index) => this.handleDeleteEntry(entry, index)}
+        chipRenderer={ChipRenderer}
+        disableUnderline
+      />
+    )
+  }
+}
+
+ChipArrayField.propTypes = {
+  /**
+   * List of entries with which to initialize the component.
+   * Note: This is not a controlled input, it's only read in componentDidMount
+   */
+  entries: PropTypes.array,
+
+  /**
+   * Whether the array should be sorted after each addition
+   */
+  sorted: PropTypes.bool,
+
+  /**
+   * The callback function called to sort the entries.
+   */
+  onSortEntry: PropTypes.func,
```

Comment:
Could detail this docs explaining that it's possible to customize how you sort methods by using this `onSortEntry` callback, such as the `onValidateEntry` docs.